packages/chatbot-server-mongodb-public/src/responsesApi.test.ts

```diff
@@ -191,6 +205,48 @@ describe("Responses API with OpenAI Client", () => {
 
       await expectValidResponses({ stream, requestBody });
     });
+    // Skipping this test for the CI as it requires a real OpenAI client, which we don't do in the CI environment
```

Comment:
I think the real client tests were working in CI when I pushed them initially! Maybe I'm just missing context here but I feel like this should work without needing to skip